### ISSUE TITLE
[Python] Inconsistent scope for `.`?

### ISSUE THREAD

**Issue by michaelblyons** [sublimehq/Packages#462]
The scope in Python for the `.` is different depending on whether it is a direct object accessor or an object accessor for a function call:

```py
foo.bar().baz
#  ^ punctuation.accessor.python source.python
#        ^ source.python
```

This is basically a duplicate of #414

**Comment by FichteFoll** [comment]
The reason for this is because expressions aren't exactly handled well at the moment. I intend to change this when I make the time to work on this refactor. Or maybe Will gets around to doing it first.

**Comment by wbond** [comment]
This should be fixed by 5ade1610c14864a02489d5dd3e9a21caef9cab75.

@FichteFoll I just homogenized the context names to use `-` as a separator. Previously some used `-`, some `_`. I've been using `-` a fair amount, and intend on standardizing on it for the default packages.

**Comment by FichteFoll** [comment]
I don't like `-` being uses since the hyphen by default is a word separator character and you can't select the entire context name with ctrl+d or double-click, which is why I changed most to use underscores.
